products/BellHybrid/CMakeLists.txt

```diff
@@ -85,6 +85,8 @@ add_hex_target(BellHybrid)
 
 include(BinaryAssetsVersions.cmake)
 include(DownloadAsset)
+include(DownloadAsset2)
```

Comment:
Could it be named more accordingly?

Reply:
yes, we need this for yesterday :) it will be merged into one with followup PR (please see linked JIRA in description )